SwiftCSV/CSV.swift

```diff
@@ -8,6 +8,8 @@
 
 import Foundation
 
+fileprivate let bom = "\u{FEFF}"
```

Comment:
I'd like to suggest two changes here:

1) Spell out `byteOrderMark` to make obvious what "bom" is supposed to mean and help other devs google the term;
2) Add a doc comment with a link to https://github.com/swiftcsv/SwiftCSV/issues/97, and/or copy parts your own super useful explanation (thanks for that!) from there.